examples/advanced_circuits_algorithms/Simons_Algorithm/Simons_Algorithm.ipynb

```diff
@@ -35,18 +35,49 @@
     "<div class=\"alert alert-block alert-info\">\n",
     "<h3>Example for n=3:</h3>\n",
     "\n",
-    "Consider the function $f:\\{0,1\\}^3\\to\\{0,1\\}^3$ defined by the truth table below.\n",
-    "\n",
-    "|$x$|   $$f(x)$$|\n",
-    "|---|------|\n",
-    "|000|\t000|\n",
-    "|001|\t001|\n",
-    "|010|\t001|\n",
-    "|011|\t000|\n",
-    "|100|\t100|\n",
-    "|101|\t101|\n",
-    "|110|\t101|\n",
-    "|111|\t100|\n",
+    "Consider the function $f:\\{0,1\\}^3\\to\\{0,1\\}^3$ defined by the truth table below.  \n",
+    "<table>\n",
+    "    <thead>\n",
+    "        <tr>\n",
+    "            <th><center>$x$</center></th>\n",
```

Comment:
I actually like what you had for the truth table earlier since it renders on GitHub. Maybe you need $$ instead of $? Ditto for some of the other centered equations in this notebook.

Reply:
Which other centered equations?

Reply:
Github seems to have issues rendering the table regardless of whether I use markdown or HTML. I tried several style fixes and markdown fixes, but I couldn't get it to work. I saw similar open issues people had commented in which math in tables does not render properly.